proposals/fh_discord.md

```diff
@@ -0,0 +1,32 @@
+# Florida Hackers Discord Chat 
+
+## Target audience
+
+<!-- Who are you expecting to impact based on your proposal? -->
+Not only will the discord improve communication for FH Organizers with permission based rooms, it also allows a more open community of Florida Hackers to be active on discord from the rest of Florida.
+
+## Community needs
+
+<!-- What specific issue are you addressing with your proposal? -->
+When it comes to determing whether or not Discord should be used over Slack, it's important to note that Slack isn't entirely free and has a user limit for the free tier. 
+
+With Discord, Florida Hackers will have the ability to grow its community without the need for limiting the amount of users in the chat. 
+
+## Resources
+
+<!-- What resources do you need to accomplish your proposal? -->
+Discord and support of FH
+<!-- How will you obtain those resources? -->
+With approvial of this proposal we can move forward
+## Media
+
+<!-- How are you preparing to deliver your community impact project? -->
+Endorsing a Florida Hackers discord at every FL hackathon starting with ShellHacks could help grow the community exponentially
+<!-- What are you going to do to prepare to deliver the content? -->
+## Timeline
+
+<!-- Develop a rough timeline for your propsal. -->
+1. Create Discord 
+2. Set up structure of discord (Permissions and whatnot)
```

Comment:
Does Discord have something like [Slackin](https://github.com/rauchg/slackin) that we could use to allow for easy requests for invitations?